index.js

```diff
@@ -1,62 +1,59 @@
-var http = require('http')
 var level = require('level-party')
+var memdb = require('memdb')
 var township = require('township')
-var appa = require('appa')
-
-var send = require('appa/send')
-var error = require('appa/error')
-var config = require('./config')
+var express = require('express')
+var bodyParser = require('body-parser')
 var hypercloud = require('./lib/cloud')
 
-var db = level(config.township.db)
-var ship = township(config.township, db)
-var cloud = hypercloud(config.cloud)
-var app = appa()
-var log = app.log
+const DAT_HASH_REGEX = /[0-9a-f]{64}$/
+
+module.exports = function (config) {
+  var db = config.township.memdb ? memdb() : level(config.township.db)
```

Comment:
Can we use `typeof db === 'string' ? level(config.db) : memdb()` or something? That memdb option is a bit confusing to me to have alongside the db option.